openapi_python_client/parser/properties.py

```diff
@@ -333,19 +333,21 @@ def get_imports(self, *, prefix: str) -> Set[str]:
         return imports
 
     @staticmethod
-    def values_from_list(values: List[str]) -> Dict[str, str]:
+    def values_from_list(values: List[Any]) -> Dict[str, Any]:
         """ Convert a list of values into dict of {name: value} """
         output: Dict[str, str] = {}
 
         for i, value in enumerate(values):
-            if value[0].isalpha():
+            if type(value) is str and value[0].isalpha():
                 key = value.upper()
             else:
                 key = f"VALUE_{i}"
             if key in output:
                 raise ValueError(f"Duplicate key {key} in Enum")
             sanitized_key = utils.fix_keywords(utils.sanitize(key))
-            output[sanitized_key] = utils.remove_string_escapes(value)
+            if type(value) is str:
```

Comment:
This can probably be joined with the case above ^^
Also we typically prefer `isinstance(value, str)` over `is type(str)` so that we can support inheritance.

Reply:
do you mean adding the string escape handling under line 341? if `value` is `'1ab"c'` then it will go into the else case, since it starts with `'1'`, though presumably they would still want to call `remove_string_escapes` on this value.

> Also we typically prefer isinstance(value, str) over is type(str) 

👍